lib/pbkit/pbkit.c

```diff
@@ -3387,33 +3387,31 @@ int pb_init(void)
     *(p++)=8;
-    pb_push(p++,NV20_TCL_PRIMITIVE_3D_SET_OBJECT8,1);
-    *(p++)=12;
-    pb_push(p++,NV20_TCL_PRIMITIVE_3D_ACTIVATE_COLORS,1);
-    *(p++)=0;
+    p=pb_push1(p,NV20_TCL_PRIMITIVE_3D_SET_OBJECT8,12);
+    p=pb_push1(p,NV20_TCL_PRIMITIVE_3D_ACTIVATE_COLORS,0);
     pb_end(p);
 
     p=pb_begin();
-    pb_push1(p,NV097_SET_FLAT_SHADE_OP,1); p+=2; //FIRST_VTX 
-    pb_push4f(p,NV097_SET_EYE_POSITION,0.0f,0.0f,0.0f,1.0f); p+=5;
-    pb_push1(p,NV20_TCL_PRIMITIVE_3D_EDGE_FLAG,1); p+=2;
-    pb_push1(p,NV20_TCL_PRIMITIVE_3D_TX_SHADER_PREVIOUS,0x00210000); p+=2; //(PSTextureInput) What previous stage is used at each stage
-    pb_push1(p,NV097_SET_COMPRESS_ZBUFFER_EN,0); p+=2; //
-    pb_push1(p,NV097_SET_SHADOW_ZSLOPE_THRESHOLD,0x7F800000); p+=2;
-    pb_push1(p,NV097_SET_ZMIN_MAX_CONTROL,1); p+=2; //CULL_NEAR_FAR_EN_TRUE | ZCLAMP_EN_CULL | CULL_IGNORE_W_FALSE
+    p=pb_push1(p,NV097_SET_FLAT_SHADE_OP,1); //FIRST_VTX 
+    p=pb_push4f(p,NV097_SET_EYE_POSITION,0.0f,0.0f,0.0f,1.0f);
+    p=pb_push1(p,NV20_TCL_PRIMITIVE_3D_EDGE_FLAG,1);
+    p=pb_push1(p,NV20_TCL_PRIMITIVE_3D_TX_SHADER_PREVIOUS,0x00210000); //(PSTextureInput) What previous stage is used at each stage
+    p=pb_push1(p,NV097_SET_COMPRESS_ZBUFFER_EN,0); //
+    p=pb_push1(p,NV097_SET_SHADOW_ZSLOPE_THRESHOLD,0x7F800000);
+    p=pb_push1(p,NV097_SET_ZMIN_MAX_CONTROL,1); //CULL_NEAR_FAR_EN_TRUE | ZCLAMP_EN_CULL | CULL_IGNORE_W_FALSE
     pb_end(p);
 
     p=pb_begin();
-    pb_push_transposed_matrix(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_A(0),pb_IdentityMatrix); p+=17;
-    pb_push_transposed_matrix(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_A(4),pb_IdentityMatrix); p+=17;
-    pb_push_transposed_matrix(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_A(8),pb_IdentityMatrix); p+=17;
-    pb_push_transposed_matrix(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_A(12),pb_IdentityMatrix); p+=17;
-/*  pb_push1(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_ENABLE(0),0x2202);  p+=2;
-    pb_push1(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_ENABLE(1),0x2202);  p+=2;
-    pb_push1(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_ENABLE(2),0x2202);  p+=2;
-    pb_push1(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_ENABLE(3),0x2202);  p+=2;
-*/  pb_push4f(p,NV097_SET_FOG_PLANE,0.0f,0.0f,1.0f,0.0f); p+=5;
-    pb_push1(p,NV20_TCL_PRIMITIVE_3D_VP_UPLOAD_CONST_ID,0x0000003C); p+=2; //set shader constants cursor at C-36
+    p=pb_push_transposed_matrix(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_A(0),pb_IdentityMatrix);
+    p=pb_push_transposed_matrix(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_A(4),pb_IdentityMatrix);
+    p=pb_push_transposed_matrix(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_A(8),pb_IdentityMatrix);
+    p=pb_push_transposed_matrix(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_A(12),pb_IdentityMatrix);
+/*  p=pb_push1(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_ENABLE(0),0x2202); 
+    p=pb_push1(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_ENABLE(1),0x2202); 
+    p=pb_push1(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_ENABLE(2),0x2202); 
+    p=pb_push1(p,NV20_TCL_PRIMITIVE_3D_CLIP_PLANE_ENABLE(3),0x2202); 
```

Comment:
These 4 lines have a space before the linebreak.

Reply:
Fixed. File seems to be riddled by lines ending with whitespace btw.